<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Upgrade</class>
 <widget class="QDialog" name="Upgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_4">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label_movie">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Search for updates</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="styleSheet">
             <string notr="true">QTableView::indicator {
    width: 24px;
    height: 24px;
}

QTableView::indicator:unchecked {
    image: url(:/images/package.png);
}

QTableView::indicator:checked {
    image: url(:/images/install.png);
}</string>
            </property>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Version</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Repository</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Size</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Peso</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QPushButton" name="back">
            <property name="text">
             <string>Back</string>
            </property>
            <property name="icon">
             <iconset resource="../images.qrc">
              <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QTextEdit" name="console">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="text">
             <string>System updated</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../images.qrc">:/images/face-smile.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="applica">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/install.png</normaloff>:/images/install.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="dettagli">
       <property name="text">
        <string>Details</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/log.png</normaloff>:/images/log.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="download">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="up_progress">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>applica</tabstop>
  <tabstop>dettagli</tabstop>
  <tabstop>back</tabstop>
  <tabstop>console</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
